samples/iterable_dataset/dataset_from_container_url.py

```diff
@@ -0,0 +1,14 @@
+"""
```

Comment:
Let's also make sure to include the copyright comment before the docstring for all of these samples.